### ISSUE TITLE
NuGet package is missing tags (keywords) and a description

### ISSUE THREAD

**Issue by andrewleader** [microsoft/WindowsAppSDK#1659]
**Describe the bug**

Internal bug: https://microsoft.visualstudio.com/OS/_workitems/edit/36840240

In NuGet, if you search for "Windows App SDK" (with spaces) or "WinAppSDK", our package comes low in the results, since we don't include those in the tags.

Additionally, if and when you find our package, it's missing a description and simply just says "WindowsAppSDK".

![image](https://user-images.githubusercontent.com/13246069/139115400-f4a83942-2750-43d3-8521-c5a568ea704d.png)

**Expected behavior**
Users should be able to find our package using common search terms, and it should have a reasonable description.

**Tags today**

```
<tags>Windows,WindowsAppSDK,WinUI,UWP,XAML,Fluent,Controls,Downlevel,Compatibility,TreeView,ColorPicker,NavigationView,MenuBar,Resources,Native,DWriteCore,Interactive Experiences,ixp</tags>
```

**Tags we SHOULD have**

(note that tags are publicly visible)...

* Adding spaces to WindowsAppSDK so searching with spaces works (no spaces is already in our title)
* Adding WinAppSDK
* Removing UWP
* Adding Project Reunion and ProjectReunion (for legacy discoverability purposes)

```
<tags>Windows,Windows App SDK,WinAppSDK,WinUI,XAML,Fluent,Controls,Downlevel,Compatibility,TreeView,ColorPicker,NavigationView,MenuBar,Resources,Native,DWriteCore,Interactive Experiences,ixp,Project Reunion,ProjectReunion</tags>
```

**Description we should have**

> The Windows App SDK empowers all Windows desktop apps with modern Windows UI, APIs, and platform features, including backwards-compatibility support.
> 
> For more information about the Windows App SDK, see https://docs.microsoft.com/windows/apps/windows-app-sdk/.

**Comment by andrewleader** [comment]
We're going to implement this in 1.0.1 in the NuGet package, and for the initial 1.0.0 timeframe we'll update the description via the online NuGet portal (just like we updated the current description for Preview 3 via the portal).

**Comment by bpulliam** [comment]
This was fixed in 1.1
